presto-tests/src/main/java/com/facebook/presto/tests/datatype/DataType.java

```diff
@@ -58,6 +60,22 @@
         return dataType(insertType, prestoResultType, DataType::quote, Function.identity());
     }
 
+    public static DataType<String> charDataType(int length)
+    {
+        return charDataType(length, "");
+    }
+
+    public static DataType<String> charDataType(int length, String properties)
```

Comment:
Having a builder would make it hard for users to define additional datatypes. Also, I'm not sure what's confusing here:
charDataType(10) -> CHAR 10
charDataType() -> CHAR
charDataType(42, "WHATEVER YOU WRITE") -> CHAR(42) WHATEVER YOU WRITE
?